spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -60,14 +60,16 @@
           },
           "creators": [{
             "person_or_org": {
-              "type": "personal",
-              "given_name": "#{user.formal_name.split(',').last}",
-              "family_name": "#{user.formal_name.split(',').first}",
-              "identifiers": {
-                "scheme": "orcid",
-                "identifier": "#{user.orcid.split('/').last}"
-              }
-            }
+               "type": "personal",
+               "given_name": "#{user.formal_name.split(',').last}",
+               "family_name": "#{user.formal_name.split(',').first}",
+               "identifiers": {
+                 "scheme": "orcid",
+                 "identifier": "#{user.orcid.split('/').last}"
+               }
+            },
+            "role": {}, # TBD on June InvenioRDM release, will contain "id": <id of role>
```

Comment:
This could probably be bounced out into it's own `let(:)` variable, but I know we are going to change this in the near future when the role field is finalized for the June release. I figure this will do for now and we can DRY things out after we know what we're dealing with.